veniq/dataset_collection/types_identifier.py

```diff
@@ -153,6 +154,7 @@ def get_lines_after_invocation(
         original_file = open(filename_in, encoding='utf-8')
         lines = list(original_file)
         lines_after_invoсation = lines[invocation_line:]
+        original_file.close()
```

Comment:
If you want, you can use `with ... as ...` construction.

Reply:
i just fixed the bug in some function which is not mine